mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/positionTooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+interface TooltipProps {
+  /**
+  * Tooltip's children (ReactNode)
+  */
+  children: ReactNode;
```

Comment:
let's use generic React.FC to get rid of children prop in the interface

Reply:
In React 18, Children are no longer part of the FC type. It seems to me that this is already redundant typification
![image](https://github.com/tritonJS826/masters-way/assets/95077794/21520238-8d76-4309-a2c9-77c68dd46e94)

Reply:
ok what about generic `PropsWithChildren` in this case? I want to avoid mentioning children in props

Reply:
Ok, done